### DISCUSSION TITLE
dcm4che with tls and ca certificate

### DISCUSSION THREAD

**Discussion by niirdan** [dcm4che/dcm4che#1349] [Q&A]
Hey Friends im trying to write the start command for storescp using TLS - KEY CRT and CA SERVER
SYNTAX IM USING :
sudo ./storescp --directory PATH OF DIR --tls KEY PATH CRT PATH --trust-store- CA PATH IP:PORT -b

I Will Be happy for an example for the exact Syntax Example , Thanks!!

**Comment by gunterze** [comment]
```
$ storescp --directory  /path/to/storedir --tls-aes -b2762 \
--key-store /path/to/key.p12 \
--key-store-pass secret-of-key.p12 \
--trust-store /path/to/ca.p12 \
--trust-store-pass secret-of-ca.p12
```
